### ISSUE TITLE
infinitive inserted before the conjugated verb

### ISSUE THREAD

**Issue by TasseDeCafe** [SekouDiaoNlp/mlconjug3#65]
Running this piece of code:

```
import mlconjug
default_conjugator = mlconjug.Conjugator(language='en')
test_verb = default_conjugator.conjugate("eat")
all_conjugated_forms = test_verb.iterate()
print(all_conjugated_forms)
```

Returns this:

`[('imperative', 'imperative present', '2s', 'eateat'), ('imperative', 'imperative present', '1p', 'eateat'), ('imperative', 'imperative present', '2p', 'eateat'), ('indicative', 'indicative past tense', '1s', 'eatate'), ('indicative', 'indicative past tense', '2s', 'eatate'), ('indicative', 'indicative past tense', '3s', 'eatate'), ('indicative', 'indicative past tense', '1p', 'eatate'), ('indicative', 'indicative past tense', '2p', 'eatate'), ('indicative', 'indicative past tense', '3p', 'eatate'), ('indicative', 'indicative present', '1s', 'eateat'), ('indicative', 'indicative present', '2s', 'eateat'), ('indicative', 'indicative present', '3s', 'eateats'), ('indicative', 'indicative present', '1p', 'eateat'), ('indicative', 'indicative present', '2p', 'eateat'), ('indicative', 'indicative present', '3p', 'eateat'), ('indicative', 'indicative present continuous', '1s 1s', 'eateating'), ('indicative', 'indicative present continuous', '2s 2s', 'eateating'), ('indicative', 'indicative present continuous', '3s 3s', 'eateating'), ('indicative', 'indicative present continuous', '1p 1p', 'eateating'), ('indicative', 'indicative present continuous', '2p 2p', 'eateating'), ('indicative', 'indicative present continuous', '3p 3p', 'eateating'), ('indicative', 'indicative present perfect', '1s', 'eateaten'), ('indicative', 'indicative present perfect', '2s', 'eateaten'), ('indicative', 'indicative present perfect', '3s', 'eateaten'), ('indicative', 'indicative present perfect', '1p', 'eateaten'), ('indicative', 'indicative present perfect', '2p', 'eateaten'), ('indicative', 'indicative present perfect', '3p', 'eateaten'), ('infinitive', 'infinitive present', 'eat', 'eateat')]`

I haven't found any other verb where this happens.

**Comment by SekouDiaoNlp** [comment]
Hi @TasseDeCafe , thanks for submitting this bug report.

I will investigate the issue over the weekend and l will let you know when I release the new version with the fix.

Thanks for using mlconjug3.

SekouDiaoNlp.

**Comment by SekouDiaoNlp** [comment]
Fixed issue #65

**Comment by SekouDiaoNlp** [comment]
Hi @TasseDeCafe  I just pushed the fix for this issue and issue #66 on the master branch here on GitHub.

I published a fix for this issue today and released a new version of mlconjug3 on PyPi.

You can install the new version by typing the following command in your terminal:
`pip install mlconjug3`

Thanks again for using mlconjug3 and for submitting bug reports.

Cheers,

SekouDiaoNlp.
